src/diceBot/YearZeroEngine.rb

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: true
+
+class YearZeroEngine < DiceBot
+  # ゲームシステムの識別子
+  ID = 'YearZeroEngine'
+
+  # ゲームシステム名
+  NAME = 'イヤーゼロエンジン'
+
+  # ゲームシステム名の読みがな
+  SORT_KEY = 'いやあせろえんしん'
+
+  # ダイスボットの使い方
+  HELP_MESSAGE = <<INFO_MESSAGE_TEXT
+・判定コマンド(YZEn+n+n)
+  YZE(能力ダイス数)+(技能ダイス数)+(修正ダイス数)  # YearZeroEngine(TALES FROM THE LOOP等)の判定(6のみ数える)
+  ※ 技能と修正ダイス数は省略可能
+INFO_MESSAGE_TEXT
+
+  ABILITY_INDEX    = 2 # 能力値ダイスのインデックス
+  SKILL_INDEX      = 4 # 技能値ダイスのインデックス
+  MODIFIED_INDEX = 6 # 修正ダイスのインデックス
+
+  setPrefixes(['\A(YZE)(\d+)(\+(\d+))?(\+(\d+))?'])
```

Comment:
```suggestion
  setPrefixes(['YZE'])
```

`\A` が入っているとシークレットダイスに反応できなくなってしまいます。
setPrefixesは `YZE` だけで大丈夫です。

ついでに、シークレットダイスのテストケースも追加をお願いします。